src/main.rs

```diff
@@ -670,18 +670,18 @@ mod tests {
         }
     }
 
-    fn prepare_virtiofsd(tmp_dir: &TempDir, cache: &str) -> (std::process::Child, String) {
+    fn prepare_virtiofsd(
+        tmp_dir: &TempDir,
+        shared_dir: &str,
+        cache: &str,
+    ) -> (std::process::Child, String) {
```

Comment:
I don't think this is related to this specific commit.

Reply:
Well they are related since I need them to pass a specific shared directory. Do you want me to separate this change in its own commit?

Reply:
Ah, right, we are using virtiofs now for our vfio test, and we need to be able to pass another directory than the vfio one. Makes sense.